plugins/katello/3.15/advanced/http_proxies.md

```diff
@@ -15,18 +15,6 @@ HTTP proxies can be created and then assigned to a product though bulk selection
 
 ## Creation
```

Comment:
Should this header be dropped and the `Creating ...` header be decreased one level?

Reply:
I considered that, too. If no objections, lets try that.